deployment/ccip/changeset/cs_ccip_home.go

```diff
@@ -1230,8 +1232,10 @@ func (c UpdateChainConfigConfig) Validate(e deployment.Environment) error {
 	if !exists {
 		return fmt.Errorf("home chain %d does not exist", c.HomeChainSelector)
 	}
-	if err := commoncs.ValidateOwnership(e.GetContext(), c.MCMS != nil, e.Chains[c.HomeChainSelector].DeployerKey.From, homeChainState.Timelock.Address(), homeChainState.CCIPHome); err != nil {
-		return err
+	if c.MCMS != nil {
```

Comment:
same. this is not accurate. Please check the ValidateOwnership func. It checks for Deployer key ownership when MCMS is false